tests/rbd_mirror/test_rbd_mirror_image_features.py

```diff
@@ -47,18 +47,18 @@ def test_image_feature(rbd_mirror, pool_type, **kw):
         poolname = config[pool_type]["pool"]
         imagename = config[pool_type]["image"]
         imagespec = poolname + "/" + imagename
-        image_feature = "object-map,fast-diff"
+        image_feature = "object-map"
```

Comment:
why `fast-diff` feature got removed?

Reply:
No particular reason, I just felt that with fast-diff feature in place the mirroring of image disable will happen easier than if we were to scan the entire image (which is the case if fast-diff feature is disabled)